src/main/java/spring/intro/config/AppConfig.java

```diff
@@ -28,8 +37,8 @@ public LocalSessionFactoryBean getSessionFactory() {
         factoryBean.setDataSource(getDataSource());
 
         Properties props = new Properties();
```

Comment:
```suggestion
        Properties properties = new Properties();
```

Reply:
Bogdan in video wrote these names, so I did the same) But ok, changed.